pkg/kubelet/kubelet.go

```diff
@@ -1823,6 +1823,13 @@ func (kl *Kubelet) syncLoop(updates <-chan kubetypes.PodUpdate, handler SyncHand
 			duration = time.Duration(math.Min(float64(max), factor*float64(duration)))
 			continue
 		}
+		if ns := kl.runtimeState.networkErrors(); len(ns) != 0 {
```

Comment:
why not combine with the above codes, like:
rs := kl.runtimeState.runtimeErrors()
rs = append(rs, kl.runtimeState.networkErrors())
if len(rs) != 0 {
    ......
}

Reply:
I've removed this lines.